tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -549,6 +550,30 @@ export class WebGPUBackend extends KernelBackend {
                 input.size < sizeThreshold);
   }
 
+  batchNormalization(
```

Comment:
We are moving towards a fully modular setup - would you mind extracting batchNormalization into its own kernel file, similar to the `Square` op? WebGL and CPU backends have not yet done this, but they will.